config/config.go

```diff
@@ -120,13 +120,6 @@ func YamlProvider() ProviderFunc {
 	}
 }
 
-// EnvProvider returns function to create environment based config provider
-func EnvProvider() ProviderFunc {
-	return func() (Provider, error) {
-		return NewEnvProvider(defaultEnvPrefix, nil), nil
-	}
-}
-
 // Environment returns current environment setup for the service
 func Environment() string {
```

Comment:
I thought unused functions were a compiler warning. Let me take a sweep and see what's left lingering